src/app/sign-up/[[...sign-up]]/page.tsx

```diff
@@ -0,0 +1,9 @@
+import { SignUp } from '@clerk/nextjs';
+
+export default function Page() {
+  return (
+    <div className='flex items-center justify-center min-h-screen'>

```

Comment:
these classes where do they come from? is this tailwind?

Reply:
yes